Makefile

```diff
@@ -192,7 +192,7 @@ mocks:
 
 .PHONY: runtests
 runtests: mocks
-	@go test ./...
+	@go test -v ./cmd/datamon//cmd
```

Comment:
there are tests other than those in `cmd`

Reply:
Yeah. Blunder. I’ve also seen it today. I unduly committed a local change just for me. Will revert

Reply:
Yeah. Blunder. I’ve also seen it today. I unduly committed a local change just for me. Will revert

Reply:
There is also an undue change in pkg/store errors: added an unused constant there.

Reply:
There is also an undue change in pkg/store errors: added an unused constant there.